Update jstz doc (#768)
jstz-dev/jstz: docs/jstzd.md

- Graceful shutdown
- Health monitoring
- Configuration settings

## Getting Started

Start the daemon:

```bash
jstzd run
```

if you see the following output, jstzd is running successfully

```

           __________
           \  jstz  /
            )______(
            |""""""|_.-._,.---------.,_.-._
            |      | | |               | | ''-.
            |      |_| |_             _| |_..-'
            |______| '-' `'---------'` '-'
            )""""""(
           /________\
           `'------'`
         .------------.
        /______________\

        0.1.0-alpha.0 https://github.com/jstz-dev/jstz

  +--------------------------------------+---------------------+
  | Address                              | XTZ Balance (mutez) |
  +======================================+=====================+
  | tz1KqTpEZ7Yob7QbPE4Hy4Wo8fHG8LhKxZSx | 60000000000         |
  +--------------------------------------+---------------------+
  | tz1TGu6TN5GSez2ndXXeDX6LgUDvLzPLqgYV | 40000000000         |
  +--------------------------------------+---------------------+
```

By default, jstzd will start on port 55555. You can customize the configuration through a JSON configuration file:

> For example to run jstzd on port 66666 and the octez node on port 8888, you can create a config.json file with the following content:

Author: todo: change

> # jstzd

Reviewer: ```suggestion
# Jstzd
```

> jstzd is a daemon service that orchestrates and manages the core components of the `jstz` environment. It serves as the central coordinator for running a jstz client node and the octez binaries (node, baker, rollup node).

Reviewer: ```suggestion
jstzd is a daemon service that orchestrates and manages the core components of the `jstz` local sandbox . It serves as the central coordinator for running a jstz client node and the octez binaries (node, baker, rollup node).
```

> The core features are:
> 
> - Automatic process spawning
> - Graceful shutdown
> - Health monitoring
> - Configuration settings

Reviewer: ```suggestion
The core features are:

- Almost instant block time and commitment period
- Pre-loaded bootstrap accounts
- Customisability through user supplied configurations
```
